src/components/breadcrumb-bar/index.tsx

```diff
@@ -39,18 +39,19 @@ function BreadcrumbBar({
       <ol className={s.listRoot}>
         {links.map(({ title, url, isCurrentPage }) => {
           const cleanTitle = title.replace(/<[^>]+>/g, '')
-          const Elem = url ? InternalLink : 'span'
+          const shouldLink = url && !isCurrentPage
+          const Elem = shouldLink ? InternalLink : 'span'
```

Comment:
🚧 I very much disagree with this change, and disagree with the design system component doing this. Can we wait to make this breaking change until we know more about why the DS team decided on this? [The APG pattern we based this component off](https://www.w3.org/WAI/ARIA/apg/patterns/breadcrumb/) of uses a link in the last item so it's reachable via keyboard/assistive technology and I'm not confident we want to break that.

Reply:
Made a task for this in 🎟️ [[breadcrumb-bar] resolve HDS spec (don't link current page item) vs WAI Authoring Practices Guide (link it)](https://app.asana.com/0/1202097197789424/1202355385649681/f) ☺️ 

I did notice the WAI guidelines seem to imply the current page being a link with `aria-current="page"` as optional:

> If the element representing the current page is not a link, aria-current is optional.

Reply:
Thank you for making the ticket! Just want to make sure we have time to discuss and document in the code/Swingset page. 😊